packages/account-v2/src/containers/POAForm/AddressDetailsForm.tsx

```diff
@@ -2,9 +2,8 @@ import React, { Fragment } from 'react';
 import { Form, Formik, FormikHelpers } from 'formik';
 import { useHistory } from 'react-router-dom';
 import { useDocumentUpload, useInvalidateQuery, useSettings } from '@deriv/api';
-import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
+import { StandaloneXmarkBoldIcon, DerivLightIcPoaLockIcon as IcPOALock } from '@deriv/quill-icons';
```

Comment:
Cant we use same name from deriv/quill-icons?

Reply:
Updated.